client/src/components/atoms/ProgressBar.tsx

```diff
@@ -0,0 +1,34 @@
+import styled from "@emotion/styled";
+
+const ProgressContainer = styled.div<{ width?: number }>(
+  ({ width = 180, theme }) => `
+  display: flex;
+  align-items: center;
+
+  progress {
+    appearance: none;
+    width: ${width}px;
+    height:12px;
+
+    ::-webkit-progress-bar {
+      border-radius: 5px;
+      background-color: ${theme.colors["blue200"]};;
+    }
+    ::-webkit-progress-value {
+      border-radius: 5px 0px 0px 5px;
+      background-color: ${theme.colors["blue400"]};
+    }
+  }
+`,
+);
+
+interface ProgressProps {
+  value: number;
+  max: number;
+}
+
+export const ProgressBar = ({ value, max }: ProgressProps) => (
```

Comment:
```suggestion
export const ProgressBar: React.FC<ProgressProps> = ({ value, max }) => (
```
처럼 `React`의 Functional Component Generic을 사용하면 조금 더 안전하고 명시적으로 작성할 수 있을 것 같네요.

맨 위에 `import React from "react";` 도 추가해줘야할거에요